CSharp/VirtoCommerce.MemberExModule.Template/VirtoCommerce.MemberExModule.Template.vstemplate

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<VSTemplate Version="3.0.0" Type="ProjectGroup" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" xmlns:sdk="http://schemas.microsoft.com/developer/vstemplate-sdkextension/2010">
+  <TemplateData>
+    <Name>Virto Commerce 2.x Member Module extension</Name>
+    <Description>A project for extending Virto Commerce member models</Description>
```

Comment:
A project for extending VirtoCommerce.Customer module with new (Supplier) member type

Reply:
Fixed